pilot/pkg/model/config_test.go

```diff
@@ -888,6 +889,54 @@ func TestIstioConfigStore_ServiceEntries(t *testing.T) {
 	}
 }
 
+func TestIstioConfigStore_Gateway(t *testing.T) {
```

Comment:
Could you please add a short comment describing the test case?

Reply:
There is a comment in the test explaining the specific timestamp stuff, `TestIstioConfigStore_Gateway` describes the general test pretty well, just like all the other tests but I can add one